### DISCUSSION TITLE
Unable to squash merge, app forcing rebasing

### DISCUSSION THREAD

**Discussion by dbut2** [community/community#73500] [Mobile]
Trying to merge pr using squash method, which is the only allowed method for the repo. Merge method for pr is reverting to rebase method even after I change to squash. As repo doesn't allow rebase I'm unable to merge at all from app.

<https://github.com/community/community/assets/61171623/2ec232a3-2e45-446e-83ad-87f36b851d0ddetails>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.137.0 (113902678)
Device: iPhone15,3 (iOS 17.0.3)
</details>

**Comment by hfhbd** [comment]
Same for me: https://github.com/community/community/assets/22521688/4f4bd8ea-897d-4d4b-9046-413b2923fa2c

**Comment by rmathewsbeyond** [comment]
Same for me as well.

**Comment by ExEr7um** [comment]
Same for me
